content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -92,8 +92,22 @@ the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
 `SupportPodPidsLimit` to work.
 {{< /note >}}
 
+## PID based eviction
+
+Eviction is the way to configure kubelet to start terminating a pod if it misbehaving.
+You can set eviction threshold via `pid.available` eviction signal
+(see [Configure Out of Resource Handling](content/en/docs/tasks/administer-cluster/out-of-resource.md)).
+You can set soft and hard eviction policies. However, if pids number growing very fast,
+node can still get into unstable state hitting the pids limit.
```

Comment:
> soft and hard eviction policies

Do you mean "soft and/or hard eviction policies"?

> if pids number growing very fast

This can be instead "if the number of PIDs is growing very fast"?